research/integrations/fedora-ci/index.md

```diff
@@ -0,0 +1,338 @@
+
+Packit should not interefere with the way the SRPM gets built.
+
+### Propagating Koji build
+
+As of now, the sources need to be already uploaded in the lookaside cache to run
+the RPM build in general (either Fedora CI or Zuul CI). Given that merges in
+dist-git are handled as fast-forward, it should be possible to just tag the
+build from the PR instead of rebuilding again (with the same sources and
+specfile).
+
+:::note
+
+This is definitely a _nice-to-have_, a follow-up.
+
+:::
+
+**TODO**:
+
+- [ ] Does it make sense to open a follow-up already, even if there's no PoC
+      yet?
+
+### RPM for Testing Farm
+
+Zuul provides its own (self-hosted) repository. Fedora CI just refers to the
+scratch build.
+
+Self-hosting the repository is not preferred, following the Fedora CI approach
+might be the best.
+
+### STI tests
+
+Older format of tests that can be run via Zuul. Based on the TF API, Testing
+Farm supports it too. Requesting STI tests from the Testing Farm should be
+possible, detection of their presence lies on Packit.
+
+**TODO**:
+
+- [ ] Should we support the STI tests?
+
+### Side tags
+
+Support for side tags has been requested in the original thread with the
+proposal. This may be required for “packaging workflows” that are more complex,
+e.g., Rust packaging (dependencies that are built as separate packages).
+
+> As a maintainer, I should be able to specify the dependencies of my package
+> that need to be verified with any proposed update.
+
+### Upstream vs Downstream
+
+By default, we sync the Packit config to the downstream. This can cause a
+conflict when Packit runs as a Fedora CI, since the config would override any
+defaults that we would've set.
+
+Additionally, there needs to be a way to distinguish between upstream/downstream
+jobs anyways, cause Testing Farm runs on the downstream install RPMs from the
+Koji rather than Copr.
+
+:::warning
+
+It would probably be easier to run stateless as the current CI, utilizing only
+defaults and ignoring any Packit config that is present.
+
+:::
+
+## User perspective
+
+### Fedora
+
+> As a maintainer I'd like to open a PR and have automated RPM build to verify
+> that, whatever I'm submitting, can be
+>
+> - built,
+> - installed, and
+> - passes the test suite.
+>
+> Adding and configuring the tests in the dist-git should be my responsibility.
+> I don't want the CI to be blocking, just annoying…
+
+**TODO**:
+
+- [ ] Packit should be able to run without any repo-specific configuration, but
+      there should be a way to override, if needed.
+
+### Fedora SIGs
+
+AFAIK the repositories are hosted on the Fedora dist-git, therefore they should
+be handled by the Fedora CI already, the experience should be similar (if not
+the same) to the regular Fedora packages.
+
+### CentOS SIGs
+
+As there is no streamlined CI for the CentOS SIGs, adoption would allow
+relatively simple integration with the Testing Farm.
```

Comment:
Thanks for covering this one as well. Would be really good to count with these from the start.